spec/components/trainees/confirmation/diversity/view_spec.rb

```diff
@@ -16,8 +14,8 @@
       end
 
       it "renders with one line to say the trainee hasn't anwered the question" do
-        expect(component.find(".diversity-information .govuk-summary-list__value")).to have_text("Not provided")
-        expect(component).to have_selector(".govuk-summary-list__row", count: 1)
+        expect(rendered_component).to have_text("Not provided")
```

Comment:
If the user has not answered the question, should it rather say: 
`expect(rendered_component).to have_text("Answer missing")`?

Reply:
This is an interesting one because in the real world this state wouldn't be valid. The user has to declare either to disclose or not disclose. I think we should remove this example altogether.